src/makeRobot.js

```diff
@@ -37,8 +38,58 @@
  *
  * @return {Robot}
  */
-function makeRobot(name, wheels, version) {
-  // write code here
+
+function makeRobot(name, wheels, chipVersion) {
+  const Robot = {
+    name: name,
+    chipVersion: chipVersion,
+    wheels: wheels,
+    coords: { x: 0, y: 0 },
+    get info() {
+      // eslint-disable-next-line max-len
+      return `name: ${this.name}, chip version: ${this.chipVersion}, wheels: ${this.wheels}`;
+    },
+    get location() {
+      return `${this.name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+    goForward(steps = 1) {
```

Comment:
```suggestion
    },
    
    get location() {
      return `${this.name}: x=${this.coords.x}, y=${this.coords.y}`;
    },
    
    goForward(steps = 1) {
```
Don't forget to add empty lines between methods to make your code easier to read